src/NativeShell/src/NativeShell.java

```diff
@@ -269,32 +289,30 @@ private boolean download() {
             callbackContext.error("error: download: " + e.getMessage());
             return false;
         }
+
         Context context = cordova.getContext();
         Uri uri = Uri.parse(url);
 
         DownloadManager.Request request = new DownloadManager.Request(uri)
-                .setTitle(title)
-                .setDescription(cordova.getContext().getString(R.string.downloading))
-                .setDestinationInExternalPublicDir(Environment.DIRECTORY_DOWNLOADS, filename)
-                .setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED);
+            .setTitle(title)
+            .setDescription(cordova.getContext().getString(R.string.downloading))
+            .setDestinationInExternalPublicDir("Jellyfin", filename)
```

Comment:
why?

Reply:
The indentation or the directory change?

Reply:
I'm not completely sure but this may clash with scoped storage I'd have to look more into this but there'll be some new restrictions on Android 11. Could you test this in an Android 11 emulator by chance?

Reply:
We should probably be using something like this then: https://developer.android.com/reference/android/provider/MediaStore